core-cli/src/main/scala/io/aiven/guardian/cli/MainUtils.scala

```diff
@@ -30,4 +40,15 @@ object MainUtils {
     promise.future
   }
 
+  // Taken from https://stackoverflow.com/a/21886322/1519631
+  def setLogbackFile(path: Path, loggerContext: ILoggerFactory): Unit =
+    Using(Files.newInputStream(path)) { inputStream =>
+      val configurator = new JoranConfigurator
+      configurator.setContext(loggerContext.asInstanceOf[Context])
```

Comment:
Yes, this `asInstanceOf` is necessary. Not sure why since it should be a subclass but otherwise Scala doesn't compile.